mullvad-relay-selector/src/relay_selector/query.rs

```diff
@@ -107,6 +108,37 @@ impl RelayQuery {
             openvpn_constraints: OpenVpnRelayQuery::new(),
         }
     }
+
+    /// The mapping from [`RelayQuery`] to all underlying settings types.
```

Comment:
What about making this a bit more explicit?
```suggestion
    /// The mapping from [`RelayQuery`] to all underlying settings types.
    /// 
    /// Useful in contexts where you cannot use the query directly but
    /// still want use of the builder for convenience. For example in
    /// end to end tests where you must use the management interface
    /// to apply settings to the daemon.
```